src/po/pt-Pt.po

```diff
@@ -0,0 +1,74 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+#, fuzzy
```

Comment:
AFAIK the fuzzy line indicates that the next translation is not to be applied. Is this intentional here or would it be better removed?

Reply:
I don't think it makes any sense here, so I'll remove it.